src/components/Question/Question.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { StyledQuestionSpan, StyledQuestionText } from "./style";
```

Comment:
Please name blocks based on their logical part, not usage one. Means `span` could be easily changed to `div` later on, so it shouldn't be part of the name. Better would be even `StyledLink`. Also you don't need to add everywhere `Question` as they are already local to this component